core/tests/protractor/embedding.js

```diff
@@ -161,6 +234,11 @@ describe('Embedding', function() {
       driver.get(
         general.SERVER_URL_PREFIX + general.SCRIPTS_URL_SLICE +
         'embedding_tests_dev_i18n_0.0.1.html');
+
+      driver.findElement(by.xpath(
+        "//input[@id='embedding-exploration-id']")).sendKeys(explorationId);
+      driver.executeScript('window.update()');
```

Comment:
Why does this need to be triggered manually? Couldn't the test hit the submit button?

(Using executeScript here is kind of "cheating".)

Reply:
adding input fields without <form> works, probably protractor has some issue with clicking on nested elements.

and it's done, i have removed 'driver.executeScript'.